eo-maven-plugin/src/main/java/org/eolang/maven/TranspileMojo.java

```diff
@@ -184,15 +184,23 @@ public void exec() throws IOException {
      * @throws java.io.IOException If any issues with I/O
      */
     private int transpile(final ForeignTojo tojo) throws IOException {
-        final int saved;
-        final Path file = tojo.verified();
+        final Path file;
+        try {
+            file = tojo.verified();
+        }  catch (final IllegalStateException exception) {
```

Comment:
@Yanich96 catching runtime exceptions is a _very_ bad practice: https://stackoverflow.com/questions/24344511/why-is-catching-a-runtimeexception-not-considered-a-good-programming-practice and https://softwareengineering.stackexchange.com/questions/91827/is-it-not-a-good-practice-to-handle-runtime-exceptions-in-the-code

Reply:
@yegor256 I thought about it and I have two solutions: 
1) creating own exception inhered from RuntimeException and catch it
2) creating own exception inhered from Exception, but it bad pattern because a lot of code will be changed due to one exception
Or do you mean another solution?